tsconfig.json

```diff
@@ -7,6 +7,5 @@
     { "path": "./apps/storybook-react-native" },
     { "path": "./apps/storybook-react" }
   ],
-  "files": [],
-  "include": ["./types"]
```

Comment:
the `"./types"` directory is left over from core which the metamask-design-system doesn't have

![Screenshot 2024-10-23 at 8 06 22 PM](https://github.com/user-attachments/assets/d9c3cf96-c495-427d-8c2f-d5acfeb4eecf)